docs/2_2_common_mechanisms.adoc

```diff
@@ -330,522 +343,711 @@ Set <<parameter,`parameters`>>, <<input,`inputs`>>, and <<start>> values, and re
+===== Clock Types
 
-_[For real-time computation use cases (e.g., in Scheduled Execution), the priority information is used also for task preemption configurations.
-It is therefore important to restrict the number of distinct priority levels for an FMU to available priority levels on the target platform and/or to avoid unnecessary computational overhead.
-A common number of different priority levels is, e.g., 100 (0 to 99), as defined in Linux based operating systems.]_
+Clocks are <<discrete>> <<fmu-variables,variables>> with a special semantic.
 
-===== Output Clocks [[outputClockVariant]]
+The following clock types are described in detail after the table.
 
-An <<outputClock>> is a general <<clock>> that ticks when a time-, state- or step-event occurs in the continuous-model partition of the FMU (for <<CommunicationPointClocks,Communication Point Clocks>> these <<outputClock>> activations can occur in all model partitions) and are identified based on their <<valueReference>>.
-The clock can be <<periodic>> or <<periodic,aperiodic>>.
+Input clocks::
+[[inputClock, `input clock`]]All clocks with <<causality>> <<input>> are collectively called <<inputClock, `input clocks`>>.
+The importer will activate <<inputClock, `input clocks`>> using <<fmi3SetClock>> or <<fmi3ActivateModelPartition>>.
+Contrary to the importer being the source of the clock activation, the definition of the clock timing comes from the FMU, either through the <<modelDescription.xml>> or calling <<fmi3GetInterval>>.
 
-_[Example: If_ latexmath:[b = x > 0] _, and a <<state event>> is defined when b changes from `false` to `true`, and b is defined as <<outputClock>>, then this <<clock>> is active whenever x changes from negative to positive values.]_
+Output clocks::
+[[outputClock, `output clock`]]There is only one <<clock>> type that is an <<outputClock>>: The <<triggered>> <<outputClock>>.
+We still refer to it as <<outputClock>> for simplicity.
+The importer calls <<fmi3GetClock>> to inquire the clock activation state.
 
-Since <<outputClock,`output clocks`>> are ticking based on model internal information, it is required in case of FMI for Co-Simulation to signal the ticking of an <<outputClock>> to the simulation environment during a <<fmi3DoStep>> or <<fmi3ActivateModelPartition>> computation.
-In most cases _[TODO: not all cases? which cases? if the time of the clock tick is a communication point already?]_ the signaling of an <<outputClock>> tick requires creating a communication point for exchanging additional information (i.e. variable values) with the simulation environment.
-Additional cases need to be handled if a <<clock>> tick is not associated to an event that requires the creation of a communication point for the model partition that generated the event. _[TODO: what?]_
++
+.Overview of Clock Types.
```

Comment:
This was done on purpose: input and output are not really defining features of clocks (especially because there is only one output clock). Separating these IO properties from the important properties that are then explained in the table was intentional. Maybe you can make a concrete proposal how we can make this clear after the merge?